src/Microsoft.SqlTools.ServiceLayer/Workspace/Workspace.cs

```diff
@@ -259,6 +298,25 @@ internal string ResolveRelativeScriptPath(string baseFilePath, string relativePa
             return combinedPath;
         }
 
+        private bool IsNonFileUri(string path)
+        {
+            try
+            {
+
+                string scheme = GetScheme(path);
+                if (scheme != null && scheme.Length > 0 && !scheme.Equals("file"))
```

Comment:
`scheme != null && scheme.Length > 0` could be shortened to `!string.IsNullOrEmpty(scheme)`